app/models/pipedrive_client.rb

```diff
@@ -0,0 +1,59 @@
+require 'pipedrive-ruby'
+class PipedriveClient
+
+  def initialize(organization)
+    #TODO Authentication mechanism based upon email and password for retrieving API token for organization
```

Comment:
Initialization method is updated for initializing client with organization as discussed.